doc/index.rst

```diff
@@ -413,6 +413,29 @@ References:
 - https://docs.gitlab.com/ce/ssh/README.html
 - https://docs.gitlab.com/ee/ci/ssh_keys/README.html
 
+(Github CI) Access to private repositories
+------------------------------------------
+
+Follow a process very similar to the Gitlab one, described above, to add public and private SSH keys to your repositories.
+
+#. Add the private SSH key to your target repository, under Settings-Secrets and Variables-Actions-Secrets.
+#. Add the public SSH key to the dependency repository, under Settings-Deploy Keys.
+#. Modify the industrial_ci yaml config, adding an action to clone the dependency as follows. The following example clones two dependencies:
+
+..  code-block:: yaml
+
+    - uses: actions/checkout@v4 # dependency1
+      with:
+        ssh-key: ${{ secrets.DEPENDENCY1 }}
```

Comment:
Is `secrets.DEPENDENCY1` a variable in the yaml file? If so don't we want to make it clear (unless it's obvious for Github Action users with basic knowledge)?

Same question to dependency2.

Reply:
OK, I'll make this a little more clear, maybe even add a screenshot. DEPENDENCY1 is the name of a "secret" that was added in the Github account settings. It's a private ssh key

Reply:
done